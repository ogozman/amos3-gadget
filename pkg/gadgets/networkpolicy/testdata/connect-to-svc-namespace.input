[
  {
    "type": "connect",
    "remote_kind": "svc",
    "port": 50051,
    "local_pod_namespace": "demo1",
    "local_pod_name": "checkoutservice-66d68cbdd-9kkkl",
    "local_pod_owner": "checkoutservice",
    "local_pod_labels": {
      "app": "checkoutservice",
      "pod-template-hash": "66d68cbdd"
    },
    "remote_svc_namespace": "demo2",
    "remote_svc_name": "paymentservice",
    "remote_svc_label_selector": {
      "app": "paymentservice"
    }
  }
]
