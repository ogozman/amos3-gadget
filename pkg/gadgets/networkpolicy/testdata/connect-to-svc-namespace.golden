apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  creationTimestamp: null
  name: checkoutservice-network
  namespace: demo1
spec:
  egress:
  - ports:
    - port: 50051
      protocol: TCP
    to:
    - namespaceSelector:
        matchLabels:
          name: demo2
      podSelector:
        matchLabels:
          app: paymentservice
  podSelector:
    matchLabels:
      app: checkoutservice
  policyTypes:
  - Ingress
  - Egress
